site/en/docs/devtools/css/issues/index.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: "layouts/doc-post.njk"
+title: "Find invalid, overridden, inactive, and other CSS"
+authors:
+  - sofiayem
+date: 2022-11-15
+description: "Discover issues with CSS properties at a glance."
+tags:
+  - css
+  - find-issues
+---
+
+This guide assumes that you're familiar with inspecting CSS in Chrome DevTools. See [View and change CSS][1] to learn the basics.
+
+## Inspect the CSS you author {: #styles }
+
+Suppose that you added some CSS to an element and want to make sure the new styles are
+applied properly. When you refresh the page, the element looks the same as before. Something is wrong.
+
+The first thing to do is [inspect the element][2] and make sure that your new CSS is actually applied to the element.
+
+Sometimes, you'll see your new CSS in the **Elements** > **Styles** pane but your new CSS is in pale text, non-editable, crossed out, or has a warning or hint icon next to it.
+
+## Understand CSS issues {: #css-issues }
+
+The **Styles** pane recognizes many kinds of CSS issues and highlights them in different ways.
+
+### Invalid {: #invalid }
+
+The **Styles** pane crosses out properties with invalid syntax and displays {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/hJxGejygnlBfMvhMtcCX.svg", alt="Warning.", width="24", height="24" %} warning icons next to them.
+
+{% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/HSQATBEOt6MholSjoji2.png", alt="Invalid CSS properties.", width="800", height="542" %}
+
+### Overridden {: #overridden }
+
+The **Styles** pane crosses out properties that are overridden by other properties according to the [Cascading order][3].
+
+{% Video src="video/NJdAV9UgKuN8AhoaPBquL7giZQo1/BNlYQHj8tpojjxyMKNDv.mp4", autoplay=false, controls=true, loop=true, class="screenshot" %}
+
+### Inactive {: #inactive }
+
+The **Styles** pane crosses out properties that are valid but have no effect because of other properties. Such properties have {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/2OyGRodvgk9neTIEAH4g.svg", alt="Information.", width="24", height="24" %} information icons next to them and are inactive because of CSS logic, not the [Cascading order][3].
```

Comment:
Which properties do `Such properties` referring to? Is that the cross-out prop or the pale prop? Suggest to make the sentence to call them out clearer.

Suggest to explain the example.